client/src/plugins/deployment-tool/StartInstanceTool.js

```diff
@@ -0,0 +1,414 @@
+    };
+
+    // (4) Trigger start instance
+    try {
+      const processInstance = await this.startWithDetails(startDetails, processDefinition);
+
+      this.showStartSuccessNotification(processInstance, startDetails);
+    } catch (error) {
+      displayNotification({
+        type: 'error',
+        title: 'Starting process instance failed',
+        content: 'See the log for further details.',
+        duration: 10000
+      });
+      log({ category: 'start-instance-error', message: error.problems || error.message });
+    }
+  }
+
+  async saveStartDetails(tab, details) {
+    const {
+      config
+    } = this.props;
+
+    const savedDetails = this.getStartDetailsToSave(details);
+
+    return config.setForFile(tab.file, START_CONFIG_KEY, savedDetails);
+  }
+
+  canStartWithDetails(details) {
+
+    const {
+      businessKey
+    } = details;
+
+    return !!businessKey;
+  }
+
+  async getSavedStartDetails(tab) {
+    const {
+      config
+    } = this.props;
+
+    return config.getForFile(tab.file, START_CONFIG_KEY);
+  }
+
+  async getSavedProcessDefinition(tab) {
+    const {
+      config
+    } = this.props;
+
+    return config.getForFile(tab.file, 'process-definition');
+  }
+
+  getStartDetailsFromUserInput(tab, details) {
+    return new Promise(resolve => {
+      const handleClose = result => {
+
+        this.setState({
+          startModalState: null
+        });
+
+        this.updateMenu();
+
+        // contract: if details provided, user closed with O.K.
+        // otherwise they canceled it
+        if (result) {
+          return resolve(result);
+        }
+      };
+
+      this.setState({
+        startModalState: {
+          tab,
+          details,
+          handleClose
+        }
+      });
+    });
+  }
+
+  startWithDetails(details, deployedProcessDefinition) {
+    const api = new CamundaAPI(details.endpointUrl);
+
+    return api.startInstance(deployedProcessDefinition, details);
+  }
+
+  showStartSuccessNotification(processInstance, details) {
+    const {
+      displayNotification
+    } = this.props;
+
+    const {
+      endpointUrl
+    } = details;
+
+    displayNotification({
+      type: 'error',
+      title: 'Process started successfully',
+      content: (CockpitLink(endpointUrl, processInstance)),
+      duration: 10000
+    });
+  }
+
+  getStartDetailsToSave(rawDetails) {
+    return pick(rawDetails, [ 'businessKey' ]);
+  }
+
+  handleFocusChange = event => {
+    const editMenu = getEditMenu(isFocusedOnInput(event));
+
+    this.updateMenu({ editMenu });
+  }
+
+  updateMenu(menu) {
+    this.props.triggerAction('update-menu', menu);
+  }
+
+  render() {
+    const {
+      startModalState
+    } = this.state;
+
+    return <React.Fragment>
+
+      <Fill slot="toolbar" group="8_deploy">
+        <Button
+          onClick={ this.startInstance }
+          title="Start Current Diagram"
+        >
+          <Icon name="play" />
+        </Button>
+        <DropdownButton
```

Comment:
Instead of having an extra button, we can go for the multi-action dropdown button in the future: https://github.com/camunda/camunda-modeler/commit/eb9a3566eb62d132b8c044ed337650ee40012ca3